### DISCUSSION TITLE
How to change Select icon globally?

### DISCUSSION THREAD

**Discussion by delowardev** [chakra-ui/chakra-ui#6007] [Q&A]
Hi there, I want to change the chevron icon of the `Select` component, I can do that by passing down the `icon` prop on every `Select` field, but how can I change it globally from the theme?

Thanks.

**Comment by shorthander** [comment]
Have you found a way to solve it?

**Comment by brad-bartlett-clear** [comment]
Also interested in this

**Comment by HaleyWardo** [comment]
I believe you can do something like this in the `extendTheme`: 

```
extendTheme({
  components: {
    Select: {
      defaultProps: {
        icon: MyCustomIcon
      }
    }
  }
}
```

**Reply by HaleyWardo** [reply]
Never mind, that wont work. It looks like `defaultProps` are only typed as 

```
type DefaultProps = {
  size?: string
  variant?: string
  colorScheme?: string
}
```

**Comment by HaleyWardo** [comment]
Alright, it looks like this ^ isn't supported. The `defaultProps` in the theme is only for theme related props. The suggestion is to create a wrapper component that has your default props and then use that throughout your app. You can follow the discussion [here](https://github.com/chakra-ui/chakra-ui/issues/1424#issuecomment-743149897).
